cosmic/microservices/cosmic-bill-viewer/values.yaml

```diff
@@ -0,0 +1,20 @@
+# Default values for cosmic-bill-viewer.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  name: missioncriticalcloud/cosmic-bill-viewer
+  tag: latest
+  pullPolicy: IfNotPresent
```

Comment:
[This PR](https://github.com/MissionCriticalCloud/bubble-toolkit/pull/249) sets `Always` as pull policy for the bubble.